src/commands/privmsg.cpp

```diff
@@ -6,15 +6,241 @@
 /*   By: faboussa <faboussa@student.42lyon.fr>      +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2024/11/04 10:18:52 by yusengok          #+#    #+#             */
-/*   Updated: 2024/11/12 11:53:58 by faboussa         ###   ########.fr       */
+/*   Updated: 2024/11/15 18:29:51 by faboussa         ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
+#include <algorithm>
+#include <iostream>
+#include <sstream>
+#include <string>
+#include <vector>
+
 #include "../../includes/Server.hpp"
 
+// PRIVMSG <target>{,<target>} <:text to be sent>
+// <target> is the nickname of a client or the name of a channel.
+
+void Server::sendPrivmsgToClient(const Client &sender, const Client &receiver,
+                                 const std::string &content) {
+  std::string message = ":" + sender.getNickname() + " PRIVMSG " +
+                        receiver.getNickname() + " " + content + "\r\n";
+  receiver.receiveMessage(message);
+}
+
+// Fonction pour valider les arguments de privmsg
+bool Server::validTargets(const std::string &arg, const Client &client,
+                          stringVector *targets) {
+  size_t commaCount = std::count(arg.begin(), arg.end(), ',');
```

Comment:
**[CHANEGED]**
Rename function to `validPrivmsgTargets` to avoid confusion because this helper function is specific for PRIVMSG but attached to Server class.